code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -186,9 +186,16 @@
 	if(istype(blobmind) && istype(C))
 		blobmind.special_role = SPECIAL_ROLE_BLOB
 		var/obj/structure/blob/core/core = new(T, 200, C, 3)
-		if(core.overmind && core.overmind.mind)
-			//core.overmind.mind.name = blob.name
-			core.overmind.mind.special_role = SPECIAL_ROLE_BLOB_OVERMIND
+		spawn(20)

```

Comment:
This looks odd, but works. The spawn(20) is needed because create_overmind has a spawn() in it which means this won't be defined if we check right away. You'd think that the later qdel(src) would cause this to fail... but it doesn't.